### PR TITLE
Merge 1.7.6.x into develop - 11/01/2020

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Merge last changes from 1.7.6.x to develop
| Type?         | improvement
| Category?     | ME
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |  No QA needed

**WARNING for reviewers**: please check this merge PR thoroughly as it seems multiple files were modified both on 1.7.6.x and develop which created git conflicts that might not have been perfectly solved.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17115)

### CODE REVIEW

File: classes/Currency.php [hunk 1]
```diff
@@ -239,10 +257,11 @@ public function __construct($id = null, $idLang = null, $idShop = null)
              * the field as a string.
              */
             if (is_array($this->symbol)) {
-                $this->localizedSymbols = $this->symbol;
+
+                $this->localizedSymbol = $this->symbol;
```
Comment (matks) [inline classes/Currency.php:261]: This is a mismatch, 1.7.6.x and develop have the same property but one adds a "s" and the other one does not

DONT MERGE BEFORE FIX

**Comment by matks** [comment]
Is it OK that
- themes/classic/assets/css/custom_rtl.css 
- themes/classic/assets/css/error_rtl.css 
- themes/classic/assets/css/theme_rtl.css 
have been deleted ? 🤔

**Comment by PierreRambaud** [comment]
@matks For RTL, yes. They mustn't be included because they are generated dynamically.

File: admin-dev/themes/default/template/header.tpl [hunk 2]
```diff
@@ -29,382 +29,384 @@
...
-		var errorLogin = '{l s='PrestaShop was unable to log in to Addons. Please check your credentials and your Internet connection.' js=1 d='Admin.Notifications.Warning'}';
-		var search_product_msg = '{l s='Search for a product' js=1 d='Admin.Orderscustomers.Feature'}';
-	</script>
+    var new_order_msg = '{l s='A new order has been placed on your shop.' js=1 d='Admin.Navigation.Header'}';
+    var order_number_msg = '{l s='Order number:' js=1 d='Admin.Navigation.Header'} ';
+    var total_msg = '{l s='Total' js=1 d='Admin.Global'} ';
+    var from_msg = '{l s='From:' js=1 d='Admin.Global'} ';
+    var see_order_msg = '{l s='View this order' js=1 d='Admin.Orderscustomers.Feature'}';
+    var new_customer_msg = '{l s='A new customer registered on your shop.' js=1 d='Admin.Navigation.Header'}';
+    var customer_name_msg = '{l s='registered' js=1 d='Admin.Navigation.Notification'} ';
```
Comment (matks) [inline admin-dev/themes/default/template/header.tpl:68]: Needs to be checked

File: src/Adapter/Customer/QueryHandler/GetCustomerForViewingHandler.php [hunk 3]
```diff
@@ -386,7 +386,7 @@ private function getCustomerProducts(Customer $customer)
 
             $viewedProducts[] = new ViewedProductInformation(
                 (int) $product->id,
-                Tools::htmlentitiesUTF8($product->name),
+                $product->name,
```
Comment (matks) [inline src/Adapter/Customer/QueryHandler/GetCustomerForViewingHandler.php:389]: Needs to be checked, seems strange to remove a sanitization step 🤔

File: admin-dev/themes/default/template/header.tpl (hunk 2 above)
Comment (matks) [reply admin-dev/themes/default/template/header.tpl:68]: I checked, it must be reversed. `develop` is right, `1.7.6.x` is wrong

File: src/Adapter/Customer/QueryHandler/GetCustomerForViewingHandler.php (hunk 3 above)
Comment (matks) [reply src/Adapter/Customer/QueryHandler/GetCustomerForViewingHandler.php:389]: It's intentional, I asked to be sure in https://github.com/PrestaShop/PrestaShop/pull/17065#issuecomment-573575948

File: classes/Currency.php (hunk 1 above)
Comment (matks) [reply classes/Currency.php:261]: OK, I confirm the same property has been added on `1.7.6.x` and `develop` by 2 different devs 😭 
- https://github.com/PrestaShop/PrestaShop/pull/14972/files#diff-bd916bf385a432f6bf6f9b61421ad52bR124
- https://github.com/PrestaShop/PrestaShop/commit/72a74d88bdd896b3e9f5248c7b050fce6f2cee21#diff-bd916bf385a432f6bf6f9b61421ad52bR108

Ping @atomiix and @jolelievre

**Comment by matks** [comment]
Closed in favor of https://github.com/PrestaShop/PrestaShop/pull/17186

**Pull request closed without merging**